src/Zynga/Framework/StorableObject/Collections/Map/V2/Importer/Storable.hh

```diff
@@ -0,0 +1,46 @@
+<?hh // strict
+namespace Zynga\Framework\StorableObject\Collections\Map\V2\Importer;
+
+use Zynga\Framework\Dynamic\V1\DynamicClassCreation;
+use Zynga\Framework\Exception\V1\Exception;
+use Zynga\Framework\StorableObject\V1\Exceptions\UnsupportedTypeException;
+
+/**
+ * Class for collection importers. This class creates object of the
+ * valueType the storableMap has specified. Since valueType is of 
+ * storable nature we expect the payload to be a key value pairs to
+ * be deserialize.
+ */
+class Storable<Tv> extends Base<Tv> {
```

Comment:
Are some of these v2 classes copies of V1? If so, would help the CR to point out such classes/blocks.